src/Constants/Cookies.js

```diff
@@ -0,0 +1,2 @@
+export const cookieName = "sessionToken";
```

Comment:
This should probably be called sessionToken as its more descriptive but will cause a few files to change